tests/late_nodes/test_delayed.py

```diff
@@ -60,6 +60,13 @@ def new_client_in_namespace(name_space, setup_bootstrap, cspec, num):
     return client_info
 
 
+# this is a path for travis's 10m timeout limit
+# we reached the timeout because epochDuration happened to be greater than 10m
+def sleep_and_print(duration):
+    time.sleep(duration/2) # wait half
+    print("Half of duration passed") # print something for travis
```

Comment:
Please print something more meaningful  (Duration of what?)

Reply:
Fixed